xbmc/addons/AddonSystemSettings.cpp

```diff
@@ -118,8 +118,12 @@ bool CAddonSystemSettings::UnsetActive(const AddonPtr& addon)
   return true;
 }
 
+bool CAddonSystemSettings::isMigratingAddons() const
```

Comment:
```suggestion
bool CAddonSystemSettings::IsMigratingAddons() const
```

always use PascalCase